release_util/management/commands/generate_history.py

```diff
@@ -116,8 +116,11 @@ def handle(self, *args, **options):
                             """.format(
                                 table=table,
                                 historical_table=historical_table,
-                                insert_columns=','.join(columns),
-                                select_columns=','.join(['t.{}'.format(c) for c in columns]),
+                                # this cmd fails for tables containing reserved keywords in column.
+                                # https://dev.mysql.com/doc/refman/5.5/en/glossary.html
+                                # Backticked columns to avoid MYSQL errors
+                                insert_columns=','.join(['`{}`'.format(c) for c in columns]),
+                                select_columns=','.join(['t.`{}`'.format(c) for c in columns]),
```

Comment:
Also, you do not need to quote identifiers if they follow the period:
```
t.order
t.`order`
```
Those BOTH work in mysql.

Reply:
Using quotes generates command as t.`order`.
Sample insert generated using this cmd that i ran locally:
```
INSERT INTO historicalabdultest(
                                `id`,`testfield`,`key`,history_date,history_change_reason,history_type,history_user_id
                            )
                            SELECT t.`id`,t.`testfield`,t.`key`,'2020-01-29','initial history population', '+', NULL
                            FROM abdultest t
                            LEFT JOIN historicalabdultest
                                ON t.id=historicalabdultest.id
                            WHERE historicalabdultest.id IS NULL
                                AND t.id >= 1
                                AND t.id < 1001
```

Raises error without it.

Reply:
Insert won't work, as `t.key` still has reserved word used without backticks.

Reply:
Are you sure?  I don't think it really matters, so do not block on this issue.

---

I tried this on the LMS read replica, and it works:

```
select t.values from site_configuration_siteconfiguration t limit 1;
```

Whereas, this fails:

```
select values from site_configuration_siteconfiguration limit 1;
```

From https://dev.mysql.com/doc/refman/5.7/en/keywords.html: 

> Exception: A word that follows a period in a qualified name must be an identifier, so it need not be quoted even if it is reserved: